src/css/LocalSvg.tsx

```diff
@@ -3,9 +3,9 @@ import { useState, useEffect, Component } from 'react';
 import type { ImageSourcePropType } from 'react-native';
 import { Platform, Image } from 'react-native';
 
-import { fetchText } from 'react-native-svg';
+import type { SvgProps } from '../elements/Svg';
```

Comment:
As for these changes, I am afraid we should leave it as it is due to this PR: https://github.com/software-mansion/react-native-svg/pull/2168. It is meant to be a different package so the imports should behave accordingly.